packages/api/src/services/emp-registry.ts

```diff
@@ -9,10 +9,10 @@ interface Config extends BaseConfig {
 }
 type Dependencies = Pick<AppState, "registeredEmps" | "provider">;
 
-export default (config: Config, appState: Dependencies) => {
+export default async (config: Config, appState: Dependencies) => {
   const { network = 1 } = config;
   const { registeredEmps, provider } = appState;
-  const address = registry.getAddress(network);
+  const address = await registry.getAddress(network);
```

Comment:
Why adding async call here? `getAddress` returns a string directly

Reply:
i think the api changed. removing core references was what this was using under the hood, and now its using `contracts-node` to provide this call, which is probably async.

Reply:
Yeah, exactly.